docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
     container_name: node1
     image: "tendermint/localnode"
     ports:
-      - "26659-26660:26656-26657"
```

Comment:
I'd rather change the prometheus port than break existing setup

Reply:
Okay I have changed it to 26665 or do you have a specific preference for which port it should be?

Reply:
not really, but better to use one from different range, say `27000`